### ISSUE TITLE
Add a toggle pronunciation button

### ISSUE THREAD

**Issue by emanuelps2708** [jzohrab/lute-v3#370]
Hi jz, I think it would be useful to have a toggle pronunciation button, the current Chinese fork has that option not only for chinese but for all languages, so I think it would be great to be added to the main lute

**Additional context**
It could help learners of languages where the writing system is not phonetic or the orthography is not very consistent

![image](https://github.com/jzohrab/lute-v3/assets/147004033/57d27940-a881-4bdc-b89d-ac345cfbf577)
![image](https://github.com/jzohrab/lute-v3/assets/147004033/f884b9c6-0690-4252-bba2-8e5bfd4d345e)
![image](https://github.com/jzohrab/lute-v3/assets/147004033/d96f24f9-3baa-4d00-8fc3-f9fb7561f562)

**Comment by jzohrab** [comment]
Yeah that's pretty sharp, it can def be helpful when the data is present. Thanks!
